retry/backoff.go

```diff
@@ -0,0 +1,10 @@
+package http_retry
```

Comment:
I'll go and update the copyright headers across the board somehwere down the line